samples/Dafny/Problem1-SumMax.dfy

```diff
@@ -0,0 +1,26 @@
+// From https://rise4fun.com/Dafny/Problem1-SumMax
+
+// This method computes the sum and max of a given array of
+// integers.  The method's postcondition only promises that
+// 'sum' will be no greater than 'max'.  Can you write a
+// different method body that also achieves this postcondition?
+// Hint: Your program does not have to compute the sum and
+// max of the array, despite the suggestive names of the
+// out-parameters.
```

Comment:
This example looks like it was taken from a tutorial. We prefer real-world samples to train our classifier. Could you provide more "realistic" use-cases?

Reply:
For sure. I assume these samples are not intended to be useful for humans to understand the language but only for the classifier? Do we have a good sense of how much code tends to provide adequate training?